lib/adf_info.py

```diff
@@ -673,20 +673,38 @@ def get_climo_yrs_from_ts(self, input_ts_loc, case_name):
 
         #Search for first variable in var_list to get a time series file to read
         #NOTE: it is assumed all the variables have the same dates!
-        ts_files = sorted(input_location.glob(f"{case_name}*.{var_list[0]}.*nc"))
+
+        #Read hist_str (component.hist_num) from the yaml file, or set to default
+        hist_str = self.get_basic_info('hist_str')
+        #If hist_str is not present, then default to 'cam.h0':
+        if not hist_str:
+            hist_str = 'cam.h0'
+        #End if
+
+        ts_files = sorted(input_location.glob(f"{case_name}.{hist_str}.{var_list[0]}.*nc"))
 
         #Read in file(s)
         if len(ts_files) == 1:
             cam_ts_data = xr.open_dataset(ts_files[0], decode_times=True)
         else:
-            cam_ts_data = xr.open_mfdataset(ts_files, decode_times=True, combine='by_coords')
+            try:
+                cam_ts_data = xr.open_mfdataset(ts_files, decode_times=True, combine='by_coords')
+            except:
+                print(" -----------    ERROR   ------------")
+                print(ts_files)
```

Comment:
Sorry, this is a remnant of me trying to do testing. I'm taking it out. I was hitting a problem where multiple time series files had been generated for case (and put in the same directory). It resulted in having multiple files identified in the search that couldn't be concatenated because they had overlapping times. Maybe that needs to be revisited sometime, but I don't think this check needs to be included here.